lib/main_common.pm

```diff
@@ -1551,7 +1551,7 @@ sub load_extra_tests_console {
     # Audio device is not supported on ppc64le, s390x, JeOS, and Xen PV
     if (!get_var("OFW") && !is_jeos && !check_var('VIRSH_VMM_FAMILY', 'xen') && !check_var('ARCH', 's390x')) {
         loadtest "console/aplay";
-        loadtest "console/soundtouch" if is_sle('<15');
+        loadtest "console/soundtouch" if is_sle('12-sp4+') && is_sle('<15');
```

Comment:
```suggestion
        loadtest "console/soundtouch" if is_sle('<12-sp4');
```
suppose to cover 12-SP5 **and** SLE15 , right ?

Reply:
should be 12 SP4 ans SP5, but I will test also 15

Reply:
in the original PR there was a test against Tumbleweed so how about just *excluding* the versions that don't work?

Reply:
It is scheduled only for sle 12 SP4 & SP5, 15 doesn't have soundtouch package (only srcpackage) http://10.100.12.155/tests/13036#step/soundtouch/6